common/beerocks/scripts/CMakeLists.txt

```diff
@@ -2,9 +2,12 @@
 if (TARGET_PLATFORM STREQUAL "linux")
     set(PLATFORM_DB_PATH ${CMAKE_INSTALL_PREFIX}/share)
     set(PLATFORM_DB_PATH_TEMP /tmp)
-elseif (TARGET_PLATFORM STREQUAL "turris-omnia")
-    set(PLATFORM_DB_PATH "/opt/prplmesh/share")
+elseif (TARGET_PLATFORM STREQUAL "openwrt")
+    set(PLATFORM_DB_PATH "/opt/prplMesh/share")
     set(PLATFORM_DB_PATH_TEMP "/tmp/beerocks")
+    if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
+        set(EXTRA_LD_PATH "/opt/intel/lib")
```

Comment:
It sucks that this is needed. Normally, cmake should set the rpath correctly so we should investigate why it doesn't happen here.

Reply:
It is not needed. I opened a PR to fix it in UGW, please remove it from prplmesh. No need for the EXTRA_LD_PATH.